src/web/backend/nextcloud/api.py

```diff
@@ -102,15 +144,22 @@ def get(self, group_name=None, action=None):
 
         if group_name is None and action is None:  # groups list
             query = request.args.get('query')
-            res = self.nextcloud.get_groups(search=query)
-            return self.nxc_response(res)
+            search = f'cn={query}*' if query else None
```

Comment:
Note for reference: It would be great if the EDAP project could hide all of the ugly `cn=...` stuff, e.g. by having a `get_group_by_name` method.

Reply:
already did some job on it - implemented `get_group`, method, but org unit is required there, should I add same method to search by cname only in posixGroup without specifying the division?

Reply:
Great, I will merge for now so I don't block you, and I will answer your question later, when I think it through.